packages/javascript/bh-shared-ui/src/components/ExploreTable/ExploreTable.tsx

```
@@ -14,109 +14,189 @@
+import { WrappedExploreTableItem } from '../../types';
+import { EntityField, format, formatPotentiallyUnknownLabel } from '../../utils';
 import NodeIcon from '../NodeIcon';
+import { ManageColumnsComboBoxOption } from './ManageColumnsComboBox';
 import { TableControls } from './TableControls';
 
-type HasData = { data?: object };
+const TABLE_CONTROLS_HEIGHT = '72px';
 
-interface ExploreTableProps<TData extends HasData> {
+const requiredColumns = REQUIRED_EXPLORE_TABLE_COLUMN_KEYS.reduce(
+    (acc, curr) => ({ ...acc, [curr]: true }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)